bugwarrior/services/phab.py

```diff
@@ -155,10 +159,14 @@ def issues(self):
 
             yield self.get_issue_for_record(issue, extra)
 
+    def revisions(self):
+        # TODO -- get a list of these from the api
+        projects = {}
```

Comment:
It's not unused, but it seems the only method used on it is `get()`.  I don't understand the idea, either, but I can delete it.  I'm trying to be a little conservative with my changes since I don't have a Maniphest install to test against, so I can't cover what is now the `tasks` method.